x11-themes/papirus-icon-theme/papirus-icon-theme-9999.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
Is this something you're actually using?

Reply:
Yeah because the git version have a fix for -bin programs and I'm actually using it

Reply:
I want a solemn promise from you that you won't adopt anything you don't use without asking me first ;)

Reply:
@thesamesam i promise, but I can assure I use these icons since 2019 (I used to use them both on manjaro and arch)

Reply:
Why not use a snapshot or add a patch? Is there no hope for a release soon? I don't really love the idea of adding a live package when there are alternatives.

Reply:
@ajakk it will be released on may, if the usual release will follow. I can drop the live package if it's a problem